examples/analysis_options.yaml

```diff
@@ -7,7 +7,7 @@ analyzer:
     implicit-dynamic: false
 
   plugins:
-    - dart_code_metrics
+  #  - dart_code_metrics
```

Comment:
:'(

Reply:
I always forget to put this plugin back.

This plugin is really causing headaches for me :(